### ISSUE TITLE
Feature Request: Package as Docker container

### ISSUE THREAD

**Issue by thickconfusion** [ipilcher/fdf#5]
Hoping to run this on a VM which has NICs on two networks. It would be most convenient if I could just spin up a container rather than building the required libraries. In addition, I considered running on Alpine, but I have even less experience building (musl-c) and installing in that environment, so an Alpine container would be even better.

**Comment by ipilcher** [comment]
I agree that it would be great to have a container image available, but I have basically no skills in the area of building such images.

I'd certainly welcome contributions in this area.

**Comment by thickconfusion** [comment]
@ipilcher Put this together for you. I chose to statically link the required library.
